BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1319,12 +1319,11 @@ public void MissileLaunch()
                 part.explosionPotential = 0; // Minimise the default part explosion FX that sometimes gets offset from the main explosion.
                 rcsClearanceState = (GuidanceMode == GuidanceModes.Orbital && hasRCS && vacuumSteerable && (vessel.InVacuum()) ? RCSClearanceStates.Clearing : RCSClearanceStates.Cleared); // Set up clearance check if missile hasRCS, is vacuumSteerable, and is in space
 
-                var mml = part.GetComponent<MultiMissileLauncher>();
-                var isClusterMissile = mml && mml.isClusterMissile;
+                bool isClusterMissile = multiLauncher && multiLauncher.isClusterMissile;
```

Comment:
This isn't triggering properly because `multiLauncher` is still `null` at this point.